pyaedt/edb_core/siwave.py

```diff
@@ -898,6 +898,13 @@ def configure_siw_analysis_setup(self, simulation_setup=None, delete_existing_se
             simsetup_info.Name = simulation_setup.setup_name
             simsetup_info.SimulationSettings.AdvancedSettings.PerformERC = False
             simsetup_info.SimulationSettings.UseCustomSettings = True
+            if simulation_setup.mesh_freq:  # pragma: no cover
```

Comment:
a unit test should be implemented for the if as well

Reply:
done